resources/js/components/filter/Options.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <div class="tw-flex tw-flex-col tw-flex-1">
+        <div class="tw-flex tw-border tw-border-gray-800 tw-mb-6">
+            <input
+                class="tw-px-2 focus:tw-outline-none tw-placeholder-gray-800 tw-font-medium tw-text-gray-800 tw-text-sm tw-py-1 tw-w-full"
+                type="text"
+                :placeholder="placeholder"
+            />
+            <div class="tw-flex tw-items-center tw-mr-2">
+                <i class="fa fa-search"></i>
+            </div>
+        </div>
+        <div
+            class="tw-bg-white tw-flex-1 tw-flex tw-flex-col md:tw-max-h-80 md:tw-min-h-80 tw-pr-3 tw-scrollbar tw-scrollbar-w-1 tw-scrollbar-track-rounded tw-scrollbar-thumb-rounded tw-scrollbar-thumb-gray-600 tw-scrollbar-track-gray-200 tw-overflow-auto"
+        >
+            <label
+                v-for="option in options"
+                :for="option.value"
+                :class="`tw-flex tw-items-center ${
+                    option.selected ? 'tw-bg-gray-200' : ''
+                } hover:tw-bg-gray-200`"
```

Comment:
Please turn this into an object like you did above

```suggestion
                :class="[
                  'tw-flex tw-items-center hover:tw-bg-gray-200',
                  {
                    'tw-bg-gray-200': option.selected,
                  }]"
```